scripts/lib/api/Pkg.ts

```diff
@@ -44,7 +44,9 @@ export class Pkg {
   constructor(kwargs: {
     name: string;
     title: string;
-    githubSlug: string;
+    // We use `null` rather than `undefined` to make sure the user really doesn't want to set this,
+    // rather than not realizing they should set it.
+    githubSlug: string | null;
```

Comment:
What effect does this have on the code's behaviour? It doesn't look like L194 distinguishes between `null` and `undefined`.

Reply:
TypeScript expects you to set `githubSlug=null` in the constructor. Otherwise you could either set `githubSlug=undefined` or simply leave off `githubSlug` entirely. I'm nervous about people leaving off `githubSlug` without realizing they shouldn't.

But maybe I'm being overly nervous actually? We now have the error for the one place where we use `githubSlug`. So it probably is fine to use `undefined`?

Reply:
Sounds good :)